src/components/microlearning/VideoDetailPage.jsx

```diff
@@ -204,7 +200,7 @@ const VideoDetailPage = () => {
                       description="Label for the original price of the course."
                     />
                   </strong>
-                  <s>${getCoursePrice(courseMetadata)} USD</s>
+                  <s>${getContentPriceDisplay(coursePrice)} USD</s>
```

Comment:
[sanity check] If the `formatPrice` is used within `getContentPriceDisplay`, does the `$` rendered here need to go away?